### DISCUSSION TITLE
ConvertTo-PnPClientSidePage : Page 'default.aspx' does not exist

### DISCUSSION THREAD

**Discussion by fnanfne** [pnp/powershell#713] [Q&A] [answered]
Hi!

I have followed some simple instructions on a blog to convert some legacy SharePoint sites to the New Experience. 

The following commands were entered in the SharePoint Online Management PowerShell client on my PC...

```
Connect-PnPOnline -Url "https://company.sharepoint.com/test" -Credentials $cred
ConvertTo-PnPClientSidePage -Identity "default.aspx" -AddPageAcceptBanner
```

However, after entering the last command, I receive the following error message...

_ConvertTo-PnPClientSidePage : Page 'default.aspx' does not exist_

I have seen a few old github posts relating to this error code but these were from 2019 and did not fix my issue.

I am on a Windows 10 PC with version 3.29.2101.0 of the SharePointPnPPowerShellOnline module is installed.

Any help on this greatly appreciated.

-fnanfne

**Comment by veronicageek** [comment, marked as answer]
The [SharePointPnPPowerShellOnline](https://github.com/pnp/pnp-powershell) module is not maintained anymore. I'd suggest you first update to the PnP.PowerShell (this repo) and try again. Here is the [documentation](https://pnp.github.io/powershell/articles/authentication.html) to get you started with the PnP.PowerShell module.

**Reply by fnanfne** [reply]
Hi veronicageek, thank you for the iniformation, appreciated, I shall get on that asap, and thx also for the documention!
